tests/Test.Microsoft.Identity.Core.UIAutomation/UserInformationFieldIds.cs

```diff
@@ -38,9 +38,22 @@ public void DetermineFieldIds(LabUser user)
         {
             if (user.IsFederated)
             {
-                // We use the same IDs for ADFSv3 and ADFSv4
-                PasswordInputId = CoreUiTestConstants.AdfsV4WebPasswordID;
-                SignInButtonId = CoreUiTestConstants.AdfsV4WebSubmitID;
+                switch (user.FederationProvider)
+                {
+                    case FederationProvider.AdfsV3:
+                    case FederationProvider.AdfsV4:
```

Comment:
how about adding ADFS 2019 both here and in MSAL?

Reply:
done